howtos/deployment.rst

```diff
@@ -0,0 +1,348 @@
+
+Then, the information collected might be used to copy the required files into the destination:
+
+.. code-block:: python
+
+    from distutils.dir_util import copy_tree
+
+    if not os.path.isdir(destination):
+        os.makedirs(destination)
+    for src_lib_dir, dst_bin_dir in dep_lib_dirs.items():
+        copy_tree(src_lib_dir, os.path.join(destination, dst_lib_dir))
+    for src_bin_dir, dst_bin_dir in dep_bin_dirs.items():
+        copy_tree(src_bin_dir, os.path.join(destination, dst_bin_dir))
+
+The advantage over the ``deploy`` generator is fine-grained control: here we copy only binaries and libraries.
+It's also could be easily modified to apply some sort of filtering (e.g. to copy only shared libraries, 
+and omit any static libraries or auxiliary files such as pkg-config .pc files).
+The extracted information may also be used to generate a simple startup script (as described below):
+
+.. code-block:: python
+
+    executable = "MyApp"  # just an example
+    varname = "$APPDIR"
+
+    def _format_dirs(dirs):
+        return ":".join(["%s/%s" % (varname, d) for d in dirs])
+
+    path = _format_dirs(bin_dirs.values())
+    ld_library_path = _format_dirs(bin_dirs.values())
+    exe = varname + "/" + executable
+
+    content = """#!/usr/bin/env bash
+    set -ex
+    export PATH=$PATH:{path}
+    export LD_LIBRARY_PATH=$LD_LIBRARY_PATH:{ld_library_path}
+    pushd $(dirname {exe})
+    $(basename {exe})
+    popd
+    """.format(path=path,
+           ld_library_path=ld_library_path,
+           exe=exe)
+
+.. note::
+
+    The full example might be found on `GitHub <https://github.com/SSE4/conan-deploy-tool>`_.
+
+Approaches
+----------
+
+system package manager
+----------------------
+
+Usually the recommended option is to use system package manager, like making **.rpm** or ***.deb** package(s). Unfortunately, 
+it's not always an option, as there are few limitations:
+
+- scalability problem - need to create one package per supported distro (likely one for `Ubuntu <https://ubuntu.com/>`_, one for `Arch Linux <https://www.archlinux.org/>`_, etc.), and formats or guidelines for each distro might differ significantly
+- it might be needed to (re)compile application N times for each of supported distro, or at least use the lowest version (see concerns about ``glibc`` below)
+- sometimes older distributions may lack packages for actual software releases (e.g. boost 1.71 might be missing in `CentOS <https://www.centos.org/?>`_ 6 or 7)
+- not all systems use package managers, for instance, some embedded systems or `LFS <http://www.linuxfromscratch.org/>`_ distributions
+
+As a result, there are other ways to help with application deployment.
+
+archives (tarballs)
+-------------------
+
+The most straightforward thing to do is to make an archive with application and all of its dependencies and resources, such as zip file, or tarball (``.tar.gz``/``.tgz``, ``.tar.bz``/``.tbz``, ``.tar.xz``/``.txz`` or just ``.tar`` file).
```

Comment:
```suggestion
You can package your shared libraries, assets, executables etc to a (``.tar.gz``/``.tgz``, ``.tar.bz``/``.tbz``, ``.tar.xz``/``.txz`` or just ``.tar`` file).
```
But I don't think we are providing any useful information here, everyone knows that something can be zipped. maybe remove enterely.

Reply:
removed